src/java.base/unix/native/libjava/ProcessImpl_md.c

```diff
@@ -74,34 +74,23 @@
  *
  * B) vfork(2): Portable and fast but very unsafe. It bypasses the memory
  *    problems related to fork(2) by starting the child in the memory image of
- *    the parent. Things that can go wrong include:
- *    - Programming errors in the child process before the exec(2) call may
- *      trash memory in the parent process, most commonly the stack of the
- *      thread invoking vfork.
- *    - Signals received by the child before the exec(2) call may be at best
- *      misdirected to the parent, at worst immediately kill child and parent.
- *
- *    This is mitigated by very strict rules about what one is allowed to do in
- *    the child process between vfork(2) and exec(2), which is basically nothing.
- *    However, we always broke this rule by doing the pre-exec work between
- *    vfork(2) and exec(2).
- *
- *    Also note that vfork(2) has been deprecated by the OpenGroup, presumably
- *    because of its many dangers.
+ *    the parent.
+ *    *** This mode is inherently dangerous, and the danger partly outside the control
+ *        of the programmer (for details, see JDK-8357090). Therefore, we deprecated
+ *        the vfork mode with JDK 25 and removed if with JDK 26. ***
  *
  * C) clone(2): This is a Linux specific call which gives the caller fine
  *    grained control about how exactly the process fork is executed. It is
  *    powerful, but Linux-specific.
  *
- * Aside from these three possibilities there is a forth option:  posix_spawn(3).
- * Where fork/vfork/clone all fork off the process and leave pre-exec work and
- * calling exec(2) to the user, posix_spawn(3) offers the user fork+exec-like
- * functionality in one package, similar to CreateProcess() on Windows.
- *
- * It is not a system call in itself, but usually a wrapper implemented within
- * the libc in terms of one of (fork|vfork|clone)+exec - so whether or not it
- * has advantages over calling the naked (fork|vfork|clone) functions depends
- * on how posix_spawn(3) is implemented.
+ * D) posix_spawn(3): Where fork/vfork/clone all fork off the process and leave
```

Comment:
I would downplay the OS launch options we don't use, only saying that vfork is dangerous (and a bit as to why) but there's no need to mention clone. And there's no need to mention the the deprecation or removal, when this is integrated its gone.